### DISCUSSION TITLE
Metro Exodus-Gold Edition地铁离去

### DISCUSSION THREAD

**Discussion by huminhang** [Gamuxorg/bbs#283] [投递和请求游戏]
游戏名称：地铁离去黄金版
游戏名称(英文)：Metro Exodus-Gold Edition
下载地址 : magnet:?xt=urn:btih:99D8073DCE8B69636FA416DFCE4EEDBF9FE26058&tr=http%3A%2F%2Fbt.t-ru.org%2Fann%3Fmagnet&dn=Metro%20Exodus%20-%20Gold%20Edition%20(1.0.6.5)%20%5Bamd64%5D%20%5BMulti%5D%20%5BSteam%5D%20%5B4A%20Engine%5D
我购买的是epic版，在发现steam出Linux版后我十分后悔。。。
这个磁链下载速度极慢，我突发奇想，用qbittorrent校验了种子和win版游戏的文件差异后，最终只需要下载1G多的文件覆盖即可，在Linuxmint20.2测试暂时正常，不过据说A卡需要mesa21.2及之后的版本，或者用专有驱动，否则有内存溢出bug。
我网盘空间有限，只上传了用于覆盖部分，完整win游戏文件可自行网上获取，注意，我是用的文件为2021年7月下载的epic版，其他版本兼容性自测。
链接：https://pan.baidu.com/s/1WV1Ph8Eo-AR2OggAziKQVw 
提取码：6nrv

**Comment by shouhuanxiaoji** [comment]
收到，尽快处理

**Comment by shouhuanxiaoji** [comment]
正在处理

**Comment by shouhuanxiaoji** [comment]
已下载。。尝试打包中。。

**Comment by shouhuanxiaoji** [comment]
处理完毕，上传中，预计下周末完成，感谢
